### DISCUSSION TITLE
If switching to Floorp builds, will your installer still work with current Waterfox?

### DISCUSSION THREAD

**Discussion by RPMozley** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#90] [Q&A] [answered]
Question for @hawkeye116477 :

If/when you switch away from Waterfox builds to Floorp (not personally interested in Floorp), will your [install-waterfox-linux](https://github.com/hawkeye116477/install-waterfox-linux) work with the current release of Waterfox (and possible future releases)?
I did try looking through the code but didn't get very far as I didn't know where to look.

**Comment by hockeymikey** [comment]
That's disappointing, Floorp looks like a joke project (and sounds like one too).

**Reply by hawkeye116477** [reply]
Did you even tried it? It's like Vivaldi with Gecko engine :smile:.
For me disappointing is that Waterfox devs very rare reply to issues at GitHub, while Floorp is more active.

**Comment by hawkeye116477** [comment]
@RPMozley I'll check that later, but probably shouldn't be problems. AppImage package will also probably remain.

**Reply by RPMozley** [reply]
I tried it out yesterday, it installed it but hung at the end (had to close it manually - unless that's what you're suppose to do?).

**Reply by RPMozley** [reply, marked as answer]
After some more installations, it seems the hang was caused by one particular version of waterfox zip archive release. All the more recent releases have not exhibited this.
